src/process/meson.build

```diff
@@ -0,0 +1,43 @@
+
+MistProcFFMPEG = executable('MistProcFFMPEG',
+                            'process_ffmpeg.cpp',
+                            join_paths(meson.source_root(), 'src/output/output_ebml.cpp'),
+                            join_paths(meson.source_root(), 'src/input/input_ebml.cpp'),
+                            join_paths(meson.source_root(), 'src/input/input.cpp'),
+                            join_paths(meson.source_root(), 'src/output/output_http.cpp'),
+                            join_paths(meson.source_root(), 'src/output/output.cpp'),
+                            join_paths(meson.source_root(), 'src/io.cpp'),
+                            header_tgts,
+                            include_directories: incroot,
+                            link_with: libmist,
+                            install: true     
+)
+
+
+MistProcMKVExec = executable('MistProcMKVExec',
+                            'process_exec.cpp',
+                            join_paths(meson.source_root(), 'src/output/output_ebml.cpp'),
+                            join_paths(meson.source_root(), 'src/input/input_ebml.cpp'),
+                            join_paths(meson.source_root(), 'src/input/input.cpp'),
+                            join_paths(meson.source_root(), 'src/output/output_http.cpp'),
+                            join_paths(meson.source_root(), 'src/output/output.cpp'),
+                            join_paths(meson.source_root(), 'src/io.cpp'),
```

Comment:
You could save a few build edges here by having MistProcFFMPEG and MistProcMKVExec share common source files as a `libcommon = static_library('common', the_common_sources)` and then having both executables link to that static library.